R/zchunk_LA124.heat.R

```diff
@@ -90,19 +90,18 @@ module_energy_LA124.heat <- function(command, ...) {
       filter(sector == "out_electricity_heat" , GCAM_region_ID %in% heat_regionIDs$GCAM_region_ID) %>%
       mutate(sector = sub("out_", "", sector)) %>%
       left_join(enduse_fuel_aggregation, by = "fuel") %>%
-      select(GCAM_region_ID, sector, year, value, heat) %>%
+      select(GCAM_region_ID,   sector, year, value, heat) %>%
       rename(fuel = heat) %>%
       group_by(fuel, sector, GCAM_region_ID, year) %>%
       summarise(value = sum(value)) %>%
       filter(fuel != is.na(fuel)) -> L124.out_EJ_R_heatfromelec_F_Yh
 
-
     # Secondary output coefficients on heat produced by main activity CHP plants
     L1231.out_EJ_R_elec_F_tech_Yh %>%
       filter(GCAM_region_ID %in% heat_regionIDs$GCAM_region_ID) %>%
       # Select only technologies that have heat output in calibrated techs mapping
       filter(technology %in% calibrated_techs$technology[calibrated_techs$secondary.output == "heat"]) %>%
-      left_join(L124.out_EJ_R_heatfromelec_F_Yh %>%
+      left_join_error_no_match(L124.out_EJ_R_heatfromelec_F_Yh %>%
                   rename(value_heatfromelec = value) %>%
```

Comment:
This is one of the output tables, so I would need to create an intermediary table with a different name or name those columns back. Might make things even more confusing. What do you think?